### PR TITLE
TST: See if percentile is breaking devdeps

### PR DESCRIPTION
**Pull request by pllim**
Attempt to understand failures in #93

### CODE REVIEW

File: lcviz/conftest.py [hunk 1]
```diff
@@ -8,7 +8,12 @@
 
 @pytest.fixture
 def helper():
-    return LCviz()
+    viz = LCviz()
+    tv = viz.app.get_viewer(viz._default_time_viewer_reference_name)
```
Comment (kecnry) [inline lcviz/conftest.py:12]:
```suggestion
    tv = viz.default_time_viewer
```

File: lcviz/conftest.py [hunk 2]
```diff
@@ -8,7 +8,12 @@
 
 @pytest.fixture
 def helper():
-    return LCviz()
+    viz = LCviz()
+    tv = viz.default_time_viewer
```
Comment (kecnry) [inline lcviz/conftest.py:12]: woops, sorry!

```suggestion
    tv = viz.default_time_viewer._obj
```

File: lcviz/conftest.py (hunk 1 above)
Comment (kecnry) [reply lcviz/conftest.py]: yep, this change was just in the latest merged PR to make viewers more flexible

**Comment by kecnry** [comment]
hmmm, can you cherrypick https://github.com/kecnry/lcviz/pull/5 here (which will break all other CI runs, but hopefully let the dev one run)?

**Comment by pllim** [comment]
I was hoping I don't have to do clone this repo locally... I guess I will have to now...

**Comment by pllim** [comment]
Okie dokie... Gonna branch from #68 instead.

**Pull request closed without merging**